features/volgindicaties.feature

```diff
@@ -0,0 +1,154 @@
+      Dan stuurt de applicatie een Av01 bericht met burgerservicenummer "999994220" naar het GBA-V
+
+   Scenario: Een gebruiker beëindigt de volgindicatie die ook voor alle andere gebruikers inactief is
+      Gegeven de persoon met burgerservicenummer "999992351" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999992351" en einddatum "2021-03-15"
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999992351" en einddatum "2021-03-01"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999992351"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999992351" met einddatum "2021-03-05"
+      Dan stuurt de applicatie een Av01 bericht met burgerservicenummer "999992351" naar het GBA-V
+
+   Scenario: Een gebruiker beëindigt de volgindicatie voor een persoon die ook door een andere gebruiker gevolgd wordt
+      Gegeven de persoon met burgerservicenummer "999992971" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999992971" en einddatum "2021-03-15"
+      En gebruiker "B" heeft een volgindicatie voor de persoon met burgerservicenummer "999992971" en einddatum "2021-06-30"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999992971"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "999992971" met einddatum "2021-03-05"
+      Dan stuurt de applicatie GEEN Av01 bericht met burgerservicenummer "999992971" naar het GBA-V
+
+   Scenario: Een gebruiker heropent een beëindigde volgindicatie
+      Gegeven de persoon met burgerservicenummer "000009945" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "000009945" en einddatum "2021-03-01"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "000009945"
+      Als gebruiker "A" op 5 maart 2021 een volgindicatie zet op de persoon met burgerservicenummer "000009945" en geen einddatum
+      Dan stuurt de applicatie een Ap01 bericht met burgerservicenummer "000009945" naar het GBA-V
+
+   Scenario: Een gebruiker heropent een beëindigde volgindicatie die eerder door een andere gebruiker gevolgd werd
+      Gegeven de persoon met burgerservicenummer "999990949" is NIET ingeschreven in deze gemeente
+      En gebruiker "A" heeft een volgindicatie voor de persoon met burgerservicenummer "999990949" en einddatum "2021-03-01"
+      En geen enkele andere gebruiker heeft een volgindicatie voor de persoon met burgerservicenummer "999990949"
+      En de applicatie heeft op 1 maart 2021 een Av01 bericht met burgerservicenummer "999990949" naar het GBA-V gestuurd
```

Comment:
Is dit relevant? Ik weet niet hoe de Update API wordt geïmplementeerd (doet eigenlijk niet toe), maar als de Update API geen actieve volgindicaties van een persoon heeft, dan kan je er toch van uit gaan dat de Update API geen volgindicatie meer heeft voor die persoon bij het GBA-V en dat het Av01 is gestuurd? Of kan het voorkomen dat na 1 maart (in dit geval) nog geen Av01 is gestuurd?

Reply:
doelgroep voor deze feature is de update applicatie, niet een client van de update API.

> Of kan het voorkomen dat na 1 maart (in dit geval) nog geen Av01 is gestuurd?
Zou niet moeten. Er kan natuurlijk ergens een storing zijn, waardoor de berichten niet konden worden gestuurd. Maar buiten dat moet de Av01 gestuurd worden op de dag van het beëindigen van de volgindicatie.
Ik heb deze conditie toegevoegd om expliciet te maken dat de volgindicatie al bij GBA-V is verwijderd, dus dat de volgindicatie nu opnieuw gezet moet worden in GBA-V